sf/draft procurement/stmt of qual.md

```diff
@@ -13,15 +13,15 @@ The Smithsonian Institution (SI) intends to release a Request for Proposal on or
 
 The bullet points below ensure that the size of the vendor is adequate for this particular work, and that the anticipated price proposal is in the same 'ballpark' as the government budget.  
 
-* Has your company have adequate size to successfully deliver the digital product (15 FTEs minimum)?
-* Does your company certify that it is able to build the product within the budgetary constraints of the Government. See size estimate of team as indicated in the draft Statements of Objectives?
+* Does your company have adequate size to successfully deliver the digital product (5-7 FTEs minimum)?
+* Does your company certify that it is able to build the product within the budgetary constraints of the Government? See size estimate of team as indicated in the draft Statements of Objectives.
 
 The bullet points below relate back to the Digital Services Playbook.  
 
 Play 1 – Understand What People Need
-  * We have identified the primary users of our service, and we conduct formal user research with these users at least monthly. These users must be real users of the product or service, and must not be a small set of pre-designated Subject Matter Experts.
-  * Over the last quarter, we have conducted for our customers/clients qualitative research methods (e.g. contextual inquiries, user interviews, on-site observations) with our customer’s/client’s users and have changed our implementation plan based on the findings.
-  * Through user research directly with end-users, we have documented and analyzed findings about user goals, needs, behaviors, and preferences, and continually refer to the findings to shape the product we and our customer/client built.
+  * We have identified the primary users of our product or service, and we conduct formal user research with these users. These users must be real users of the product or service, and must not be a set of pre-designated Subject Matter Experts.
+  * We have recently conducted qualitative research (e.g. contextual inquiries, user interviews, on-site observations) for our customers/clients including their users, and have iterated on our implementation plan based on the findings.
```

Comment:
- We adjusted to be inclusive of products and services
- We adjusted the measure of time to be less prescriptive